library/core/src/tuple.rs

```diff
@@ -100,6 +100,26 @@ macro_rules! tuple_impls {
                 }
             }
         }
+
+        #[stable(feature = "array_tuple_conv", since = "1.63.0")]
```

Comment:
Since looks wrong here and lower?

Reply:
Oh, indeed! This should be using the new `CURRENT_RUSTC_VERSION` placeholder.

Reply:
The PR was made before `CURRENT_RUSTC_VERSION`, oops 😅 

I'll make a fixup PR

Reply:
Opened #111213